pom.xml

```diff
@@ -12,11 +12,14 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.mule.modules</groupId>
     <artifactId>mule-json-module</artifactId>
-    <version>2.5.0-SNAPSHOT</version>
+    <version>2.5.0</version>
```

Comment:
Remember that this project uses the runtime pipeline, so if you merge this PR to the master branch (or any other protected branch), the release process will be triggered.

Reply:
We can change the version back to a snapshot so that the release can be performed in a separate PR.